test/SlowTests/Server/Replication/ReplicationTombstoneTests.cs

```diff
@@ -329,87 +331,19 @@ public async Task Replication_of_document_should_delete_existing_tombstone_at_de
                 var tombstonesAtStore2 = GetTombstones(store2);
                 Assert.Empty(tombstonesAtStore2);
 
-                stats = await store2.Maintenance.SendAsync(new GetStatisticsOperation());
+                stats = await GetDatabaseStatisticsAsync(store2);
                 Assert.Equal(0, stats.CountOfTombstones);
             }
         }
 
-        [Fact]
-        public async Task Replication_of_document_should_delete_existing_tombstone_at_destination_sharding()
```

Comment:
have you deleted this test? why?

Reply:
yep, cause we have the sharded version now in `Replication_of_document_should_delete_existing_tombstone_at_destination`